inspect_builder_test.go

```diff
@@ -95,79 +99,55 @@ func testInspectBuilder(t *testing.T, when spec.G, it spec.S) {
 							"io.buildpacks.buildpack.order",
 							`[{"group": [{"id": "buildpack-1-id", "optional": false}, {"id": "buildpack-2-id", "version": "buildpack-2-version-1", "optional": true}]}]`,
 						))
-					})
```

Comment:
I compressed all these tests into one simpler test that asserts on the whole structure.